doc/HttpLuaModule.wiki

```diff
@@ -3563,7 +3563,7 @@ This method does not work in HTTP/2 requests yet.
 == ngx.req.get_method ==
 '''syntax:''' ''method_name = ngx.req.get_method()''
 
-'''context:''' ''set_by_lua*, rewrite_by_lua*, access_by_lua*, content_by_lua*, header_filter_by_lua*, balancer_by_lua*''
+'''context:''' ''set_by_lua*, rewrite_by_lua*, access_by_lua*, content_by_lua*, header_filter_by_lua*, balancer_by_lua*, log_by_lua*''
```

Comment:
For completeness' sake, we should also note that this API works in `body_filter_by_lua*` as well.

Reply:
OK. I will also note this.